packages/blocks/src/__internal__/content-parser/parse-markdown.ts

```diff
@@ -0,0 +1,215 @@
+import { assertExists } from '@blocksuite/global/utils';
+import type { Page } from '@blocksuite/store';
```

Comment:
the parser of markdown and notion maybe can use plugins. Try to reduce duplicate code。